.github/workflows/check-links.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: Check links
+
+on: [pull_request]
+
+jobs:
+  markdown-link-check:
+    name: Check links using markdown-link-check
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks out a copy of your repository on the ubuntu-latest machine
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Check links on new changes
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          config-file: ".github/linters/mlc_config.json"
+          check-modified-files-only: "yes"
```

Comment:
So following some recent discussions, I feel that we may now probably want to run this checker on a regular basis, links can just disappear suddenly and it would be nice to get notified of that using a scheduled check like https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onschedule. And an appropriate report, maybe via an issue automatically opened?

If you don't want to include this here, we can record an issue in the template repo and use it where relevant (for documentation at leasT)

Reply:
Opened https://github.com/EGI-Foundation/documentation/issues/157 but I don't think it's needed here for the time being.